README.md

```diff
@@ -82,6 +83,17 @@ docker run -p 8000:8000 wagtailcms
 
 ```
 
+## Environment variables
+
+| Variable        | Default       | Description                       |
+| --------------- | :-----------: | --------------------------        |    
+| DBHOST         | unistatsdb.postgres.database.azure.com            | DB host url/string                |
+| DBPORT         | 5432          | DB connection port                |
+| DBNAME         | unistatsdevdb        | db name to use                    |
+| DBUSER         | unistatsmanager@unistatsdb           | DB user                           |
+| DBPASSWORD     | 8f555f62-6009-4a77-a1aa-75658ac2bc1a              | DB password                       |
+
+
 ### Contributing
 
 See [CONTRIBUTING](CONTRIBUTING.md) for details.
```

Comment:
You've linked to Contributing guidelines and license but these files don't currently existed.

I suggest adding them to the root level of the repository